src/_includes/common/filter-events.md

```diff
@@ -0,0 +1,130 @@
+
+#### Sampling
+
+If a sample rate is defined for the SDK, the SDK evaluates whether this event should be sent as a representative fraction of events. 
+
+**Note:** The SDK sample rate is not dynamic; changing it requires re-deployment. In addition, setting an SDK sample rate limits visibility into the source of events. Setting a rate limit for your project may better suit your needs.
+
+When you enable sampling in your SDK, you choose a percentage of collected errors to send to Sentry. For example, to sample 25% of your events:
+
+{{ include.filter-sample-rate_content }}
+<!-- Guideline: add a code sample that supports sampling at 25% example for the SDK your are documenting -->
+
+For Sentry's Performance features (which are currently in Beta), we **strongly recommend**  sampling your data for two reasons. First, though capturing a single trace involves minimal overhead, capturing traces for every single page load, or every single API request, has the potential to add an undesirable amount of load to your system. Second, by enabling sampling you’ll more easily prevent yourself from exceeding your organization’s [event quota](/accounts/quotas/), which will help you manage costs.
+
+When choosing a sampling rate, the goal is to not collect *too* much data, but to collect enough data that you are able to draw meaningful conclusions. If you’re not sure what rate to choose, start with a low value and gradually increase it as you learn more about your traffic patterns and volume, until you’ve found a rate which lets you balance performance and cost concerns with data accuracy.
+
+### Using Hints
+
+Hints are available in two places: 
+
+1. `beforeSend` / `beforeBreadcrumb` 
+2. `eventProcessors` 
+
+Event and Breadcrumb `hints` are objects containing various information used to put together an event or a breadcrumb. Typically `hints` hold the original exception so that additional data can be extracted or grouping can be affected. 
+
+For events, those are things such as `event_id`,  `originalException`,  `syntheticException` (used internally to generate cleaner stack trace), and any other arbitrary `data` that you attach. 
+
+For breadcrumbs, the use of `hints` is implementation dependent. For XHR requests, the hint contains the xhr object itself; for user interactions the hint contains the DOM element and event name and so forth.
+
+In this example, the fingerprint is forced to a common value if an exception of a certain type has been caught:
+
+{{ include.filter-hint_content }}
+<!-- Guideline: add a code sample that supports the SDK your are documenting -->
+
+#### Hints for Events
+
+`originalException`     The original exception that caused the Sentry SDK to create the event. This is useful for changing how the Sentry SDK groups events or to extract additional information.
```

Comment:
Are all these whitespaces here (and below) on purpose?

Reply:
Between the term and its definition? Not specifically. Thanks for pointing it out (FYI -  I have planned time at the end of the final phase to complete format fixes that pop up, but am glad you pointed this out here).